docs/bucket/notifications/README.md

```diff
@@ -1,6 +1,8 @@
 # MinIO Bucket Notification Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
 
-Events occurring on objects in a bucket can be monitored using bucket event notifications. Event types supported by MinIO server are
+Events occurring on objects in a bucket can be monitored using bucket event notifications. If you are planning to use this capability, please be aware that when used as a  Gateway (except NAS gateway) you  will not be able to use bucket notifications.
+
+Event types supported by MinIO server are
```

Comment:
This line explicitly mentions that it is only supported by `MinIO server`.

Reply:
Any chance that we can be more explicit? I think it does not hurt and it might save some time for potential users in the future.